packages/driver/src/cy/commands/navigation.ts

```diff
@@ -1138,6 +1138,8 @@ export default (Commands, Cypress, cy, state, config) => {
           // throw an error, else we'd be in a endless loop,
           // we also need to disable retries to prevent the endless loop
           if (previouslyVisitedLocation) {
+            // _retires is `private`. We're not using `retries()` method here because it breaks some tests.
+            // @ts-ignore
```

Comment:
For some reason, `retries()` method doesn't work here [for some tests](https://app.circleci.com/pipelines/github/cypress-io/cypress/40975/workflows/dbb89f4b-9b26-444a-83e9-33937f703753/jobs/1693561). I used `//@ts-ignore` to avoid this failure.